css/main.css

```diff
@@ -762,3 +762,354 @@ footer.col-sm-12 {
 .map-sites div h4 {
   padding-left:15px;
 }
+
+    .bs-slider{
```

Comment:
can you make sure that this starts at the beginning of the line?
Otherwise looks good, thanks!